### ISSUE TITLE
Conversion crash for ins1485195

### ISSUE THREAD

**Issue by 20DM** [HEPData/hepdata-converter#60]
Hi,

I noticed an issue with the conversion for `CMS_2017_I1485195`, which prints an error message:
```
 ERROR:hepdata_converter.writers.array_writer:TypeError encountered when parsing --2.2 and 3.3
```
for me locally.

This is coming from the downward-error of the FONLL column in [Table 7](https://www.hepdata.net/record/ins1485195) where they have put an extra minus sign in front of the digit: `--2.2`. (We should probably fix that...?)

What confuses me a little is that when I try to export to YODA using a web interface, I get an html with the error log saying it encountered a `KeyError` when trying to access the asymmetric errors for that bin. However, when I run the conversion locally from the yaml inputs, it does not crash. It would still print the error from the `array_writer` I pasted above, but then still manages to produce a YODA file where it just doesn't put any error for the affected value.

On the other hand, when using the web interface it must have expected an up/down error for that bin, tried to access the `up`/`down` keys when they didn't exist and crashed. Do you understand why the behaviour is different?

**Comment by GraemeWatt** [comment]
Thanks for pointing out the mistake in [Table 7 of ins1485195](https://www.hepdata.net/record/ins1485195?version=1&table=Table%207).  I've replaced `--2.2` by `-2.2` and the YODA2 download is working now.  The different behaviour is because you made a separate PR #54 to skip ill-defined error components after the main PR #53 that added YODA2 support.  The deployed `hepdata-converter` package corresponds to the last release [0.3.0](https://github.com/HEPData/hepdata-converter/tree/0.3.0) from 6th November 2023 that includes PR #53 but not PR #54.  I'll try to release and deploy an update fairly soon, although I don't see that it is urgent now that I've fixed the few records that required patches to the YODA2 converter.
